packages/sshnoports/bundles/shell/install.sh

```diff
@@ -71,6 +84,10 @@ install_single_binary() {
   mkdir -p "$dest"
   cp "$script_dir/$1" "$dest/$1"
   echo "Installed $1 to $dest"
+  if is_root & ! [ -f "$user_bin_dir/$1" ] ; then
+    ln -sf "$dest/$1" "$user_bin_dir/$1"
+    echo "Linked $user_bin_dir/$1 to $dest"
+  fi
```

Comment:
If the binaries were installed as root, we need to do a symbolic to where the binaries were installed, since we install the binaries to /usr/local/bin if we are root